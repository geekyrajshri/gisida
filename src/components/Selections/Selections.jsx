import React from 'react';
import PropTypes from 'prop-types';

require('./Selections.scss');

const Selections = ({
    UIfilters
}) => (
        <div className="filter-selection">
            <div className="pre-text">YOUR SELECTIONS: </div>
            <div className="selections">{UIfilters.join(', ')}</div>
        </div>
)

export default Selections;