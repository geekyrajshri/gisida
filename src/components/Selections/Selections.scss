.filter-selection {
    position: fixed;
    top: 4.5em;
    left: 19.5%;
    min-width: 30%;
    width: auto;
    max-width: 70%;
    padding: 8px;
    box-shadow: 0 1px 7px rgba(0, 0, 0, 0.2);
    font-weight: 600;
    font-size: 12px;
    background: white;
    .pre-text {
        color: dimgray;
    }
    div {
        display: inline;
    }
    .selections {
        color: #cb181d;
        display: inline;
    }
}