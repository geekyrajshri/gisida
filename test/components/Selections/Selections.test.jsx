import React from 'react';
import renderer from 'react-test-renderer';
import Selections from '../../../src/components/Selections/Selections';

describe('Selections', () => {
  const UIfilters = ['test_filters'];
  it('Selections renders filter selections', () => {
    const component = renderer.create(
      <Selections
        UIfilters={UIfilters}
      />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
