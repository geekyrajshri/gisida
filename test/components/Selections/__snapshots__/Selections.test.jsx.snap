// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selections Selections renders filter selections 1`] = `
<div
  className="filter-selection"
>
  <div
    className="pre-text"
  >
    YOUR SELECTIONS: 
  </div>
  <div
    className="selections"
  >
    test_filters
  </div>
</div>
`;
